<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1051</width>
      <height>970</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>20</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Player Information</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>117</width>
       <height>106</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="LifeLabel">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Life:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LifeValue">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="EnegyLabel">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Enegy:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="EnegyValue">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>200</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="pressinfo">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>200</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>300</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionChoose_Map"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionGaming_Setting"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChoose_Map">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Choose Map</string>
   </property>
  </action>
  <action name="actionGaming_Setting">
   <property name="text">
    <string>Gaming Setting</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
